### PR TITLE
Added Kyrgyz language.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: django/conf/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -230,6 +230,10 @@ msgstr ""
 msgid "Korean"
 msgstr ""
 
+#: conf/global_settings.py:106
+msgid "Kyrgyz"
+msgstr ""
```
Comment (Soyuzbek) [inline django/conf/locale/en/LC_MESSAGES/django.po:235]: msgstr "Кыргызча"

File: django/conf/locale/__init__.py [hunk 2]
```diff
@@ -332,6 +332,12 @@
         'name': 'Korean',
         'name_local': '한국어',
     },
+    'ky': {
+        'bidi': False,
+        'code': 'ky',
+        'name': 'Kyrgyz',
+        'name_local': 'Кыргыз тили',
```
Comment (Soyuzbek) [inline django/conf/locale/__init__.py:339]: 'name_local': 'Кыргызча'

Comment (claudep) [reply django/conf/locale/__init__.py:339]: Thanks, I'll fix it!

File: docs/releases/3.1.txt [hunk 3]
```diff
@@ -292,7 +292,7 @@ Internationalization
   (string) value to explicitly state that the cookie is sent with all same-site
   and cross-site requests.
 
-* Added support and translations for the Algerian Arabic language.
+* Added support and translations for the Algerian Arabic and Kyrgyz languages.
```
Comment (felixxm) [inline docs/releases/3.1.txt:295]: Maybe we should use `Kirghiz`.

Comment (Soyuzbek) [reply docs/releases/3.1.txt:295]: [https://en.wikipedia.org/wiki/Kyrgyz_people](Kyrgyz_people, wiki), It would raise a lot of misunderstanding and chaos.

Comment (felixxm) [reply docs/releases/3.1.txt:295]: Just asking, because `en-US` spelling checker prefers `Kirghiz`.

**Comment by felixxm** [comment]
Thanks both :+1:

**Pull request merged**